### ISSUE TITLE
Why OCNLI fall into `中文文本分类`？

### ISSUE THREAD

**Issue by zhimin-z** [flageval-baai/FlagEval#26]
感觉划分到`选择问答`也make sense。。

**Comment by xuanricheng** [comment]
Indeed, normally it might be considered as '中文推理', but at the moment it hasn't been categorized that finely. Moreover, this problem is evaluated in a manner similar to classification, which is why it was categorized under '中文文本分类. Future changes to this categorization might be possible.

**Comment by zhimin-z** [comment]
> Indeed, normally it might be considered as '中文推理', but at the moment it hasn't been categorized that finely. Moreover, this problem is evaluated in a manner similar to classification, which is why it was categorized under '中文文本分类. Future changes to this categorization might be possible.

感谢回复，但是我想确定一下，选择题跟分类题难倒有什么本质区别吗？ @xuanricheng

**Comment by xuanricheng** [comment]
> > Indeed, normally it might be considered as '中文推理', but at the moment it hasn't been categorized that finely. Moreover, this problem is evaluated in a manner similar to classification, which is why it was categorized under '中文文本分类. Future changes to this categorization might be possible.
> 
> 感谢回复，但是我想确定一下，选择题跟分类题难倒有什么本质区别吗？ @xuanricheng

我们根据任务内容和形式大致的分成选择问答和文本分类

**Comment by zhimin-z** [comment]
> > > Indeed, normally it might be considered as '中文推理', but at the moment it hasn't been categorized that finely. Moreover, this problem is evaluated in a manner similar to classification, which is why it was categorized under '中文文本分类. Future changes to this categorization might be possible.
> > 
> > 
> > 感谢回复，但是我想确定一下，选择题跟分类题难倒有什么本质区别吗？ @xuanricheng
> 
> 我们根据任务内容和形式大致的分成选择问答和文本分类

两者的区别在哪里？我感觉选择题跟分类题在机器学习领域本质上是一类问题
